web/client/plugins/Measure.jsx

```diff
@@ -42,6 +42,7 @@ const selector = (state) => {
         withReset: isOpenlayers(state),
         showExportToGeoJSON: isOpenlayers(state),
         showAddAsAnnotation: showAddAsAnnotationSelector(state) && isOpenlayers(state),
+        trueBearing: isTrueBearingEnabledSelector(state),
```

Comment:
Please document this option in JSDoc (`@prop {object}`) and sub  linking also the ISO. both (.measureTrueBearing and .fractionDigits, configurations have to be explainded)